### PR TITLE
Monokai theme for ui-rework branch

### PR DESCRIPTION
**Pull request by barnybug**
Rework of #162 for the ui-rework branch.

### CODE REVIEW

File: bundles/monokai-themes/init.lua [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  howl.ui.theme.unregister 'Monokai'
+end
+
+return {
+  info = {
+    author = 'Monokai themes adapted by Barnaby <barnaby at pickle.me.uk>',
+    description = [[
+      The Monokai themes original designed for Textmate.
+
+      Adapted for the howl editor by Barnaby Gray.
```
Comment (shalabhc) [inline bundles/monokai-themes/init.lua]: Micro-nit: capitalize Howl?

File: bundles/monokai-themes/monokai.moon [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+embedded_bg  = '#484848'
+
+-- General styling for context boxes (editor, command_line)
+content_box = {
+  background:
+    color: background
+
+  border:
+    width: 1
+    color: border_color
```
Comment (shalabhc) [inline bundles/monokai-themes/monokai.moon]: I get a white border - border_color undefined?

File: bundles/monokai-themes/monokai.moon [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+      color: base1
+
+    color: grey
+    font: bold: true
+    padding: 1
+}
+
+return {
+  window:
+    :background
```
Comment (shalabhc) [inline bundles/monokai-themes/monokai.moon]: I got a transparent window once - does this need `color: background`?

**Comment by barnybug** [comment]
Fixes made

File: bundles/monokai-themes/monokai.moon [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+  border_bottom:
+    width: 3
+    color: border_color
+
+  header:
+    background:
+      color: grey_darkest
+
+    border_bottom:
+      color: base1
```
Comment (shalabhc) [inline bundles/monokai-themes/monokai.moon]: base1 also undefined?

**Comment by barnybug** [comment]
Thanks, caught a couple of others too.

**Comment by nilnor** [comment]
All right, the ui-rework stuff is in master now. As mentioned in a  commit comment previously, the final change that happened was that the gutter styling stanza changed, so that needs to be updated here as well.

**Comment by barnybug** [comment]
Theme updated.

**Comment by shalabhc** [comment]
Sorry for being off topic, but why do I see @kirbyfan64  and my conversation from c799f21fd4d here? Very confusing.

**Comment by barnybug** [comment]
ah - the pull request request was based off the old ui-rework branch, doesn't appear to be a way to change that. I'll create a new pull request from master.

**Pull request closed without merging**